src/app/findImageUrl.js

```diff
@@ -0,0 +1,7 @@
+export function findImageUrl(name) {
+    console.log('correct answer name', name)
+    function emptySpacesToUnderscore(name) {
+        return `swapi-json-server/public/${name.split(' ').join('_')}.jpg`
```

Comment:
nie jestem do końca pewny czy to co jest w folderze swapi-json-server jest dla nas do wykorzystania... mamy obrazki w static/assets/ z podziałem na kategorie ale tam są tylko numerki, dlatego mówiliśmy o zmapowaniu jednego na drugie na spotkaniu chyba

Reply:
ok, widzę że w tym swapi.dev indexy ludzi zgadzaja sie z nazwa (numerem) zdjecia, także zamiast name wrzuciłbym id i wyciagnął zdjecie z static/assets

Reply:
naprawione